util/Migrator/DbScripts/2022-07-11_00_Initial_Organization.sql

```diff
@@ -0,0 +1,547 @@
+IF OBJECT_ID('[dbo].[OrganizationPasswordManager]') IS NOT NULL
+
+BEGIN
+    DROP TABLE [dbo].[OrganizationPasswordManager]
+END
+GO
+
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [dbo].[OrganizationPasswordManager](
+    [Id] [uniqueidentifier] NOT NULL,
+    [OrganizationId] [uniqueidentifier] NOT NULL,
+    [Plan] [nvarchar](50) NOT NULL,
+    [PlanType] [tinyint] NOT NULL,
+    [Seats] [int] NULL,
+    [MaxCollections] [smallint] NULL,
+    [UseTotp] [bit] NULL,
+    [UsersGetPremium] [bit] NOT NULL,
+    [Storage] [bigint] NULL,
+    [MaxStorageGb] [smallint] NULL,
+    [MaxAutoscaleSeats] [int] NULL,
+    [RevisionDate] DATETIME NULL
+) 
+GO
+
+IF OBJECT_ID('[dbo].[OrganizationSecretsManager]') IS NOT NULL
```

Comment:
Same here, please move the create script into this block but check first, `IS NULL`.

Reply:
Sure thing!